src/main.ts

```diff
@@ -12,41 +12,19 @@ async function run(): Promise<void> {
   await setPaths()
 }
 
-interface GHAsset {
-  name: string
-  url: string
-  browser_download_url: string
-}
-
-interface GHAssets {
-  assets: GHAsset[]
-}
-
-async function getLatestRTXAssetURL(): Promise<string> {
-  const output = await exec.getExecOutput(
-    'curl',
-    ['-sSf', 'https://api.github.com/repos/jdxcode/rtx/releases/latest'],
-    {silent: true}
-  )
-  const json: GHAssets = JSON.parse(output.stdout)
-  const platform = `${getOS()}-${os.arch()}`
-  const asset = json.assets.find(a => a.name.endsWith(platform))
-  if (!asset) {
-    const assets = json.assets.map(a => a.name).join(', ')
-    throw new Error(`No asset for ${platform}, got: ${assets}`)
-  }
-  return asset.browser_download_url
-}
-
 async function setupRTX(): Promise<void> {
   const rtxBinDir = path.join(os.homedir(), '.local/share/rtx/bin')
+  const platform = `${getOS()}-${os.arch()}`
   await fs.promises.mkdir(rtxBinDir, {recursive: true})
-  await exec.exec('curl', [
-    '-sSfL',
-    '--compressed',
+  await exec.exec('gh', [
```

Comment:
frankly it would be way easier to use rtx.pub but as of right now I'm footing the bill personally for that so this helps me keep costs down by making gh pay for the bandwidth

Reply:
if I can manage to get a megacorp to pay for rtx.pub then we could put it there and wouldn't need to set the GITHUB_TOKEN, but for now we can use this